apps/gitness/src/pages-v2/pull-request/pull-request-compare.tsx

```diff
@@ -287,16 +340,16 @@ export const CreatePullRequest = () => {
   })
 
   useEffect(() => {
-    if (pullReqData?.number && pullReqData.title && pullReqData.description) {
+    if (pullReqData?.number && pullReqData.title) {
       setPrBranchCombinationExists({
         number: pullReqData.number,
         title: pullReqData.title,
-        description: pullReqData.description
+        description: pullReqData?.description || 'No description available'
```

Comment:
Can't the `description` just be empty?

Reply:
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/ba2916f2-b0ae-434e-8159-4af159a2780b" />
i just thought it looked weird when theres no description
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/5ec11c9e-15ef-492c-978b-b57da11a4418" />
@vardanbansal-harness

Reply:
Agreed, though feeling that it's more of a product question.